### PR TITLE
Create installable package

### PR DESCRIPTION
**Pull request by nvlaarhoven**
Make repo ready to upload to pypi.

Followed instructions here https://packaging.python.org/tutorials/packaging-projects/

### CODE REVIEW

File: csv_wrangler/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '0.1.3'
+__version__ = '2.0.0'
```
Comment (RealOrangeOne) [inline csv_wrangler/__init__.py]: We should just bump this to `1.0.0`, no need to do 2 major version bumps

File: setup.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 name = 'csv-wrangler'
 package = 'csv_wrangler'
-description = 'Statically-typed CSV helpers'
+description = 'Statically typed Python 3 CSV generation helpers'
```
Comment (RealOrangeOne) [inline setup.py]: I _think_ this should still by hyphenated?

File: README.md [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
---------
+## Getting Started
 
-Currently not on `pip`, as this isn't quite ready to hit the big time.  Whack this into your requirements file:
+### Installation
 
-```
-    git+git://github.com/dabapps/csv-wrangler.git@v0.1.3#egg=csv-wrangler
-```
+Install from PIP
```
Comment (RealOrangeOne) [inline README.md]: nitpick: You install _with_ `pip`. You install _from_ PyPI

File: csv_wrangler/__init__.py (hunk 1 above)
Comment (nvlaarhoven) [reply csv_wrangler/__init__.py]: Woops, you're right

File: setup.py (hunk 2 above)
Comment (nvlaarhoven) [reply setup.py]: sure

File: README.md (hunk 3 above)
Comment (nvlaarhoven) [reply README.md]: Very nitpick indeed, copied this from `django-db-queue`, but I do like correctness

**Pull request merged**